Source/Core/DolphinQt2/Debugger/MemoryViewWidget.cpp

```diff
@@ -129,14 +132,14 @@ void MemoryViewWidget::Update()
         hex_item->setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable);
         const u32 address = addr + c * (16 / GetColumnCount(m_type));
 
-        if (PowerPC::memchecks.OverlapsMemcheck(address, 16 / GetColumnCount(m_type)))
+        if (is_mram && PowerPC::memchecks.OverlapsMemcheck(address, 16 / GetColumnCount(m_type)))
           hex_item->setBackground(Qt::red);
         else
           row_breakpoint = false;
 
         setItem(i, 2 + c, hex_item);
 
-        if (PowerPC::HostIsRAMAddress(address))
+        if ((!is_mram || PowerPC::HostIsRAMAddress(address)))
```

Comment:
Not sure if it's the diff being a little wonky, but I don't think the extra parentheses are necessary here.